include/pybind11/detail/internals.h

```diff
@@ -140,17 +140,23 @@ struct type_info {
 /// Tracks the `internals` and `type_info` ABI version independent of the main library version
 #define PYBIND11_INTERNALS_VERSION 2
 
+#if defined(_DEBUG)
+#   define PYBIND11_BUILD_TYPE "debug"
```

Comment:
preceding underscore? (``_debug``)

Reply:
Yeah, thats MSVC specific.
And i suspect that whole problem might only appear with MSVC anyways

Reply:
ah. That does make more sense.